apps/nextjs-website/src/components/organisms/Auth/ConfirmLogin.tsx

```diff
@@ -0,0 +1,111 @@
+'use client';
+import { translations } from '@/_contents/translations';
+import {
+  Box,
+  Grid,
+  Card,
+  Snackbar,
+  Alert,
+  Link,
+  Stack,
+  Typography,
+  TextField,
+  Divider,
+  Button,
+} from '@mui/material';
+import { IllusEmailValidation } from '@pagopa/mui-italia';
+import { useCallback, useState } from 'react';
+
+interface ConfirmLoginProps {
+  onBackStep: () => null;
+  confirmLogin: (code: string) => Promise<void>;
+}
+
+const ConfirmLogin = ({ onBackStep, confirmLogin }: ConfirmLoginProps) => {
+  const {
+    auth: { confirmlogin },
+    shared,
+  } = translations;
+
+  const [error, setError] = useState<string | null>(null);
+  const [code, setCode] = useState<string>('');
+
+  const onConfirmLoginHandler = useCallback(() => {
+    confirmLogin(code).catch((e) => setError(e.message));
```

Comment:
The user has only one tentative to send the code. So on failure we should not allow another tentative

Reply:
Here's the activity: [DEV-1128](https://pagopa.atlassian.net/browse/DEV-1128)